lib/gds_api/rummager.rb

```diff
@@ -14,6 +14,30 @@ def search(args, additional_headers = {})
       get_json(request_url, additional_headers)
     end
 
+    # Perform a search, returning the results as an enumerator.
+    #
+    # The enumerator abstracts away rummager's pagination and fetches new pages when
+    # necessary.
+    #
+    # @param args [Hash] A valid search query. See Rummager documentation for options.
+    # @param page_size [Integer] Number of results in each page.
+    #
+    # @see https://github.com/alphagov/rummager/blob/master/docs/search-api.md
+    def search_enum(args, page_size: 100, additional_headers: {})
```

Comment:
Why does `page_size` default to `100` here?

Reply:
Seemed like a reasonable default.